docs/UserGuide.md

```diff
@@ -408,61 +408,72 @@ For undoing a clear command:
 
 * Clear all contacts.
 
-![edit format](images/undo-UG/after_clear.png)
+![undo format](images/undo-UG/after_clear.png)
 
 * Undo.
 * Message shown to the user: "Undo Successful! ALl contacts have been added back!".
 * GUI reflects that all cleared contacts are added back.
 
-![edit format](images/undo-UG/after_clear_undo.png)
+![undo format](images/undo-UG/after_clear_undo.png)
 
 For undoing an edit command:
 
 * Edit a contact.
 
-![edit format](images/undo-UG/after_edit_1.png)
+![undo format](images/undo-UG/after_edit_1.png)
 
 * Undo.
 * Message shown to the user: "Undo Successful! Reverted back to: X", where X are the details of the person before
   the edit.
 * GUI reflects that the edited contact has been reverted.
 
-![edit format](images/undo-UG/after_edit_undo.png)
+![undo format](images/undo-UG/after_edit_undo.png)
 
 #### Precise expected outputs on failure:
 * Caused by using undo when there are no previous commands to undo.
 * Error message shown to the user: "There is no command to undo!".
 * GUI reflects undo text in red font.
 
 
-![edit format](images/undo-UG/undo_fail.png)
+![undo format](images/undo-UG/undo_fail.png)
 
 ### Sorting contacts : `sort`
 
-Sort contact lists by parameter name or appointment date.
+Sort contact lists by prefix name or appointment date.
 
-Precise command format: `undo parameter`
+Format: 
+* `sort PREFIX`
 
 Example commands:
 * `sort n/`
+* `sort appt/`
 
-Acceptable parameters:
-* `n/` sort by Name parameter
-* `appt/` sort by Appointment Date parameter
+Acceptable prefix tag:
+* `n/` sort by Name prefix
+* `appt/` sort by Appointment Date prefix
 
-![edit format](images/sort-UG/sortformat.png)
+![sort format](images/sort-UG/sortformat.png)
 
 Precise expected output on success:
 * Successful addition message. 'X person listed!', where X are the number of clients in the address book.
 * The sorted entry is displayed in the address book GUI.
 
-![edit format](images/sort-UG/sortresult.png)
+![sort format](images/sort-UG/sortresult.png)
 
 Precise expected outputs on failure:
-* If a required parameter is missing, an error message should indicate the invalid format. 
-* If a parameter provided is invalid (e.g., e/), an error message should indicate the invalid format.
+<br> If a required parameter is missing, an error message should indicate the invalid format. 
+<br> If a parameter provided is invalid (e.g., e/), an error message should indicate the invalid format.
```

Comment:
should combine these into one line (i.e If a require parmeter is missing or invalid, an error message should...)
alternatively, can have two screenshots and messages for each. but thats a bit repetitive so best to stick to combining them.  would be cleaner!

Reply:
noted fixed! but it might look very long